docs/advanced.md

```diff
@@ -0,0 +1,59 @@
+# Advanced usage
+
+## Using SyncTeX
+
+JupyterLab LaTeX supports using SyncTeX to map a location in the
+text editor to a location in the generated PDF, as well as the reverse.
+To reveal the page of the PDF correspoding to the cursor in the text editor,
+right-click in the editor and select "Show ".
+To reveal the location in the text editor corresponding to a page of the PDF,
+right click on the PDF and select "Scroll PDF to Cursor"
+
+SyncTeX generates its mapping during the compilation of the `.tex` document
```

Comment:
For developers (not users) can you describe a bit more about what is in the `.synctex.gz` file? That's what I was intending to ask for in the previous case. I have not found good resources on understanding the file and how it is to be processed and that's going to be crucial for other people being able to contribute to this code.

Reply:
I think it's an opaque binary file, not intended to be human-readable or extendable.

Reply:
ugh… gross. then can you explicitly state that? And then describe how we're using it,  specifically that you either pass in line and column numbers or you pass in page and x, y coords and get back the other. Discussing what the x,y coords mean (which I know I've asked about elsewhere).

Reply:
Looking a bit closer: the synctex output is automatically gzipped, since it can be on the large side for large pdfs. If you turn off gzip, it looks human-readable, but for the most part I think it is a private format meant for use by the `synctex` command line executable.